src/test/java/api/ItemsByInstanceResourceTest.java

```diff
@@ -77,6 +78,38 @@ void canGetInstanceById() {
       hasJsonPath("tenantId", is(TENANT_ID_UNIVERSITY)))));
   }
 
+  @Test
+  void canGetEmptyResult() {
+    IndividualResource instance = instancesFixture.basedUponDunkirk();
+    UUID instanceId = instance.getId();
```

Comment:
can we inline it?

Reply:
You mean like this?
UUID instanceId = instancesFixture.basedUponDunkirk().getId();
Since we do not call instance object, we could do like this

Reply:
yes, you use this variable only once

Reply:
updated, thanks!